hooks/conan-center.py

```diff
@@ -634,11 +634,17 @@ def test(out):
 
     @run_test("KB-H060", output)
     def test(out):
+        ext_to_be_checked = [".cmake", ".conf", ".cfg", ".diff", ".md", ".patch", ".py", ".txt",
+                             ".yml", ".am", ".xml", ".json", ".in", ".ac", ".tsx", ".tmx"
+                             ".proto", ".capnp", ".c", ".cc", ".c++", ".cpp", ".cxx", ".c++m",
+                             ".cppm", ".cxxm", ".h++", ".hh", ".hxx", ".hpp", ".qrc", ".pro", ".build"]
         recipe_folder = os.path.dirname(conanfile_path)
         for root, _, files in os.walk(recipe_folder):
             if os.path.relpath(root, recipe_folder).replace("\\", "/").startswith("test_package/build"):
                 continue
             for filename in files:
+                if not any(filename.endswith(ext) for ext in ext_to_be_checked):
```

Comment:
I don't know there's a rule disallowing uppercase characters in filenames?
```suggestion
                if not any(filename.lower().endswith(ext) for ext in ext_to_be_checked):
```

Reply:
No, good catch!